### ISSUE TITLE
Create a homepage site

### ISSUE THREAD

**Issue by dk1844** [AbsaOSS/login-service#1]
Like other AbsaOSS projects, a public doc page should exists for login gateway at https://absaoss.github.io/
This site should be also referenced in publish.sbt and propagated with publish metadata.

**Comment by TheLydonKing** [comment]
Shouldn't this be integrated into the Info endpoint? Does it make sense to make them separate endpoints?

**Comment by dk1844** [comment]
No, a subsite at https://absaoss.github.io/ is meant. Take a look there how other AbsaOSS projects are presented there.

**Comment by TheLydonKing** [comment]
> No, a subsite at https://absaoss.github.io/ is meant. Take a look there how other AbsaOSS projects are presented there.

Oh, I see. Ok, perfect, Understood.

**Comment by TheLydonKing** [comment]
This has been completed:
https://absaoss.github.io/info-login-service
